src/app/share/views/SharedLinksView/hooks/useSharedContextMenu.ts

```diff
@@ -71,6 +73,8 @@ const useSharedContextMenu = ({
 
     const getItemContextMenu = (item: AdvancedSharedItem) => {
       const userUUID = item?.user?.uuid;
+      const isNotRootFolderAndIsNotViewerUser = !isCurrentUserViewer && !isRootFolder;
+      const isRootFolderAndIsOwner = isRootFolder && isItemOwnedByCurrentUser(userUUID);
```

Comment:
Perhaps better name for the constant would be `isSubFolderAndEditorUser`, like the second one. 
Adding `not` to the names adds some complexity to the reading that may confuse the reader. 
Better to use statements whenever possible.